api/src/main/scala/com/avast/clients/rabbitmq/api/RabbitMQManualConsumer.scala

```diff
@@ -0,0 +1,20 @@
+package com.avast.clients.rabbitmq.api
+
+import scala.language.higherKinds
+
+trait RabbitMQManualConsumer[F[_], A] {
+
+  /** Retrieves one message from the queue, if there is any.
+    *
+    * @return Some(DeliveryHandle[A]) when there was a message available; None otherwise.
+    */
+  def get(): F[Option[DeliveryWithHandle[F, A]]]
+}
+
+/** Trait which contains `Delivery` and it's _handle through which it can be *acked*, *rejected* etc.
+  */
+trait DeliveryWithHandle[F[_], A] {
```

Comment:
Anyone has better idea how to name it? :-)

Reply:
Is this necessary? I would think that I would use the consumer to `ack`/`reject` messages and I woud think that this could be called `Message` or `Delivery` as you already have so I think this could even be removed altogether.

Reply:
Solved in person.